go.work

```diff
@@ -0,0 +1,6 @@
+go 1.21.1
+
+use (
+	.
+	./test/testground
+)
```

Comment:
[question] are there any tips/tricks/gotchas contributors should know about using Go workspaces?

Reply:
ideally we'd use local references to everything, then they're way cooler. unfortunately when we have to reference a pushed version of code, like in a docker container, they kinda suck since there's always a chicken and egg problem.

for example we push v1.5.0 of the app, then we import v1.5.0 in the testground workspace, which would require cutting a new release if it was being used by anyone lol